core/src/main/java/com/yahoo/imapnio/async/data/QResyncSeqMatchData.java

```diff
@@ -0,0 +1,61 @@
+package com.yahoo.imapnio.async.data;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+
+/**
+ * This class models the QRESYNC Sequence match data.
+ */
+public class QResyncSeqMatchData {
+    /** Message sequence number. */
+    private List<MessageNumberSet> messageSeqNumbers;
+
+    /** Corresponding UIDs. */
+    private List<MessageNumberSet> uids;
+
+    /**
+     * Constructor.
+     * @param messageNumberSet message sequence numbers
+     * @param uid UIDs
+     */
+    public QResyncSeqMatchData(@Nonnull final List<MessageNumberSet> messageNumberSet, @Nonnull final List<MessageNumberSet> uid) {
+        this.messageSeqNumbers = messageNumberSet;
+        this.uids = uid;
+    }
+
+    /**
+     * Get the message sequence number.
+     * @return message sequence numbers
+     */
+    public List<MessageNumberSet> getMessageSeqNumbers() {
+        return messageSeqNumbers;
+    }
+
+    /**
+     * Get the uid numbers.
+     * @return UIDs
+     */
+    public List<MessageNumberSet> getUids() {
+        return uids;
+    }
+
+    /**
+     * Convert to command line string.
```

Comment:
this is a toString() method, which will be called for other usages (like object concatenation in logger), should not be used for command line string.  please rename the method to something else.